docs/pages/serving_an_origin.mdx

```diff
@@ -55,123 +57,309 @@ Once you go through the process, locate your credential files and set the follow
+
+> Environment variable configuration does not support the same complex structures that can be built with yaml configuration, such as lists.
 
 The first time the origin is started, you will see something that looks like the following:
 
 ```console
-$ pelican origin serve -f osg-htc.org -v $PWD:/demo
+$ pelican origin serve -f https://osg-htc.org -v $PWD:/demo
 
 Pelican admin interface is not initialized
 To initialize, login at https://localhost:8444/view/initialization/code/ with the following code:
 551220
 ```
+See the [browser configuration](#login-to-admin-website) documentation section for more information about initializing your origin's web interface.
+
+### Origin and Namespace Capabilities
+
+Capabilities are the configuration options you can assign to origins and namespace prefixes to determine what kinds of access controls you want them to respect. In the previous yaml configuration, we configured the origin capabilities using the `Capabilities` list of the `Origin.Exports` block by specifying that the federation prefix `/your/federation/prefix` supports "Reads" and "Writes". This list of capabilities can be used for further control of what types of operations the namespace is willing to support. Available capabilities include:
+
+- "Reads": When included, objects from the namespace can be read with a valid authorization token.
+- "PublicReads": When set, objects from the namespace become public and require no authorization to read.
+- "Writes": When included, objects can be written back to the storage backend by Pelican. Write operations _always_ require a valid authorization token.
+- "DirectReads": When included, a namespace indicates that data being pulled from it should only be pulled through a cache and not directly by clients. This may be useful in cases where the origin isn't very performant or has to pay egress costs when data moves through it. Note that this is respected by federation central services, but may not be respected by all clients.
+- "Listings": When included, the namespace indicates it will allow object discovery. Be careful when setting this for authorized namespaces, as this will allow anyone to discover the names of objects exported by this namespace.
+
+> **NOTE:** Most origins should have either "Reads" or "PublicReads" enabled. If neither is set, the origin won't export any data.
+
+There is an important distinction between _origin_ capabilities and _namespace_ capabilities. While it's sometimes easy to treat origins and namespaces as the same thing, Pelican must distinguish between them because two separate origins may export portions of the same namespace, and a single origin may export two disparate prefixes. The only exception to this rule is when a single origin serves a single namespace.
+
+To configure _origin_ capabilities, you can set top-level options for the origin:
```

Comment:
The description in `parameter.yaml` seems to contradict what was stated here: 

> NOTE: This config option is incompatible with multiple exports defined via `Origin.Exports` and requires that the origin exports
  only a single path.